i3lock-fancy

```diff
@@ -77,6 +79,7 @@ while true ; do
 	    convert -list font | awk -F: '/Font: / { print $2 }' | sort -du | command -- ${PAGER:-less}
 	    exit 0 ;;
 	-n|--nofork) i3lock_cmd+=(--nofork) ; shift ;;
+        -u|--nounlockindicator) i3lock_cmd=(i3lock -i "$image" -u) ; shift ;;
```

Comment:
Why are you resetting the command instead of `i3lock_cmd+=(-u)`?

Reply:
Why *not*? The only differences seem to me to be improved readability, and avoiding repetitive options if some user script ends up passing "-u -u -u -u".

Reply:
It's possible someone can have flags which are then over written, such as:
    i3lock-fancy -gp -f Noto-Sans -u

Reply:
@meskarune : All the options that you mentioned are handled differently by the script, so nothing would be overwritten. Wait a minute, the no-fork option does conflict, so I do need to make the change.